tests/test_tensor.py

```diff
@@ -0,0 +1,234 @@
+from __future__ import annotations
+
+import random
+
+import pytest
+import torch
+from helpers import calc_mat_vec_wavefunction, get_op_support, random_pauli_hamiltonian
```

Comment:
I still believe this is exactly what `@fixture` are for.

Reply:
Tbh I am not sure I fully understand fixtures. It's just that to me they are "fixed" things that are reused in the setup of the tests, and these are instead functions that act on some input and produce some output which is different every time.

The `random_pauli_hamiltonian` I can see how it could be made a fixture, but not the other two.